lib/routemaster/config.rb

```diff
@@ -47,7 +47,10 @@ def hosts
     end
 
     def cache_expiry
-      Integer(ENV.fetch('ROUTEMASTER_CACHE_EXPIRY', 86_400 * 365))
+      # Do not increase this default value. It's likely that cached data will include PII
+      # and 90 days is the maximum permitted retention period at Deliveroo. A higher value
+      # means we would need to worry about purging caches at the end of the period.
+      Integer(ENV.fetch('ROUTEMASTER_CACHE_EXPIRY', 86_400 * 90))
```

Comment:
Because you can still choose to override it at your own discretion. If people need to override the value then they can do so, but it'll put a greater burden on them in terms of PII handling to be able to purge data after the retention period (and prove they're doing so).